### PR TITLE
Landing page for `MicroOVN` documentation.

### PR DESCRIPTION
**Pull request by mkalcok**
Initial text for documentation landing page.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -2,7 +2,20 @@
 MicroOVN
 ========
 
-See `Example product documentation`_ for how to set up this landing page.
+``MicroOVN`` is a snap-based distribution of a ``OVN (Open Virtual Network)``.
```
Comment (fnordahl) [inline docs/index.rst:5]: s/of a/of/

File: docs/index.rst [hunk 2]
```diff
@@ -2,7 +2,20 @@
...
 
-For quick help on RST format, see the :ref:`cheat-sheet`. More information is
-available in the `reStructuredText style guide`_.
+It allows users to easily deploy ``OVN`` cluster with just a few commands.
```
Comment (fnordahl) [inline docs/index.rst:7]: deploy an ...

File: docs/index.rst [hunk 3]
```diff
@@ -2,7 +2,20 @@
...
+Aside from regular ``OVN`` packages, ``MicroOVN`` comes bundle with cli utility
```
Comment (fnordahl) [inline docs/index.rst:8]: s/bundle/bundled/
s/cli/CLI/

File: docs/index.rst [hunk 4]
```diff
@@ -2,7 +2,20 @@
...
+it allows adding or removing cluster members and status checking.
+
+Besides the ease of deployment and convenient CLI, another benefit of
+``MicroOVN`` is its self-contained nature. It is distributed as a strictly
+confined snap which means that it can be easily upgraded/downgraded/removed
+without affecting host system.
+
+``MicroOVN`` can be useful for wide range of users. It lowers a barrier of
+entry to ``OVN`` for people that are not yet familiar with it by automating as
+much of a deployment process as possible. On the other hand the aim is for
```
Comment (fnordahl) [inline docs/index.rst:19]: s/On the other hand/At the same time, /

**Review by fnordahl — CHANGES_REQUESTED** [review]
a few suggestions for grammar/typos, otherwise looks like a great start to me!

**Comment by mkalcok** [comment]
Thanks for the review. I updated the grammar and typos.

**Review by fnordahl — APPROVED** [review]
Lgtm, thanks!

**Pull request merged**